### ISSUE TITLE
MyriaRelation should support indexing

### ISSUE THREAD

**Issue by dhutchis** [uwescience/myria-python#55]
In the [MyriaRelation](https://github.com/uwescience/myria-python/blob/master/myria/relation.py#L14) class, there should be support for indexing. Indexing into a MyriaRelation is part of the example iPython notebook and seems like a very useful feature. Instead I see the following:

`TypeError: 'MyriaRelation' object does not support indexing`

**Comment by shrjain** [comment]
It would be good to have indexing, but you can do the following for now maybe
MyriaRelation(dataset['relationKey']).to_dict()

**Comment by dhutchis** [comment]
Yep, that and the rest of the examples work fine. So this is a small note. The indexing should be added or the example notebook should be changed.
